composer.json

```diff
@@ -23,12 +23,7 @@
         "php": "^5.6 || ^7.0",
         "ext-curl": "*",
         "ext-json": "*",
-        "opentracing/opentracing": "1.0.0-beta5@dev",
-        "psr/http-message": "^1.0",
-        "psr/log": "^1.0.0",
-        "symfony/polyfill-php55": "^1.9",
-        "symfony/polyfill-php56": "^1.9",
-        "symfony/polyfill-php70": "^1.8.0"
+        "opentracing/opentracing": "1.0.0-beta5@dev"
```

Comment:
https://github.com/DataDog/dd-trace-php/pull/206/files#diff-b5d0ee8c97c7abd7e3fa29b9a27d1780L40-L43 Should probably also be removed

Reply:
👍 removed, thanks